nap/cache/flask_cache.py

```diff
@@ -1,16 +1,13 @@
-from __future__ import unicode_literals
-from __future__ import absolute_import
 try:
     import flask_caching
 except ImportError as e:
-    raise ImportError("Error loading Flask-Caching: %s" % e)
+    raise ImportError('Error loading Flask-Caching: %s'.format(e))
```

Comment:
this still has the %s, you should change to {} or {0}. you could also change to an f string.